proposals/WoT Architecture Explainer.md

```diff
@@ -7,127 +7,48 @@ This section header seems chatty and redundant.  The opening can just be the ope
+It defines the abstract architecture and spans the design space for current and future so-called building blocks, which are specified in separate documents.
+Specifically, the purpose of the WoT Architecture document is to provide
+
+- the basic terminology,
+- use cases and derived requirements the W3C WoT Architecture should satisfy,
+- definitions of the basic concepts and architectural constraints,
+- an overview over the WoT building block documents and their interplay, and
+- a guide for mapping the abstract architecture onto a variety of concrete deployment scenarios
 
-[//]: # (MK: How will the web be better when this feature launches? And who will it help?)
+The architecture document lays the foundation for the WoT building block specifications:
 
-[//]: # (McCool: it is pretty hard to cleanly divide this into goals and non-goals... maybe this should be one section, "Goals and Non-Goals"?)
+- the <a href="https://w3c.github.io/wot-thing-description/">WoT Thing Description</a> (REC Track),
+- the <a href="https://w3c.github.io/wot-binding-templates/">WoT Binding Templates</a>,
+- the <a href="https://w3c.github.io/wot-scripting-api/">WoT Scripting API</a>, and
+- the <a href="https://w3c.github.io/wot-security/">WoT Security and Privacy Considerations</a>.
 
-For WoT in general the goal is to support existing standards
-and make them easier to use and integrate.
+Of these, the WoT Security and Privacy Considerations provide a general analysis that cross-cuts the other building blocks, each of which also include their own specific security and privacy considerations.
+
+## Goals
 
-The WoT consolidates IoT metadata in WoT Thing Descriptions
-to foster interoperability in two ways:
-1. By enabling and enhancing machine-to-machine communication, as devices can
-   read WoT Thing Descriptions and use the information contained in them
-   to automatically adapt to details of a particular protocol, using
-   the interaction/protocol binding abstraction.
-2. By serving as a common, uniform format for developers to document and
-   retrieve all details necessary to access an IoT service and make use of that service's data.
+* Simplify the integration of different IoT devices and platforms.
+* Complement existing IoT standards and platforms.
+* Build upon the Web architecture, in particular URIs, registered representation formats (media types), and hypermedia.
+* Allow for IP-based IoT protocols other than HTTP (given they fulfill the Uniform Interface constraint of REST).
+* Allow for different integration patterns: at the device level, gateway/edge node, and the cloud.
 
-To better support semantic interoperability, WoT Thing Descriptions may
-optionally make use of domain-specific vocabularies,
-for which explicit extension points are provided.
+[//]: # MK: Leaving out Semantic Web aspects such as Linked Data vocabularies -- needs to be aligned with spec content.
```

Comment:
@mkovatsc Does that mean you want to add semantic web aspects to the architecture document?

Reply:
It is an overall goal of the Web of Things to enable semantic interoperability, but at the same time not to force users to implement Semantic Web aspects. This is mainly taken care of the TD spec, but still an overall goal, hence architecture.

Since there is no text on it so far, I did not just mention it under goals for now.